src/frequenz/sdk/actor/power_distributing/power_distributing.py

```diff
@@ -538,6 +538,24 @@ def _get_components_pairs(
 
         return bat_inv_map, inv_bat_map
 
+    def _get_working_batteries(self, batteries: Set[int]) -> Set[int]:
+        """Get subset with working batteries.
+
+        If none of the given batteries are working, then treat all of them
+        as working.
+
+        Args:
+            batteries: requested batteries
+
+        Returns:
+            Subset with working batteries or input set if none of the given batteries
+            are working.
```

Comment:
The second line needs to be indented, to be compatible with out documentation tool.